executable/build.go

```diff
@@ -119,8 +200,11 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 	}
 
 	cp := []string{context.Application.Path}
-	if s, ok := m.Get("Class-Path"); ok {
-		cp = append(cp, strings.Split(s, " ")...)
+	cp = append(cp, allJarFiles...)
```

Comment:
Do we need to add all those jars? Wouldn't they be already include in the executable jar manifest?

Reply:
In addition, as mentioned above, the allJarsFiles contains only jars that are directly in directory, it wouldn't include `./lib`.

Reply:
> In addition, as mentioned above, the allJarsFiles contains only jars that are directly in directory, it wouldn't include `./lib`

Can you give me a reference where there would be a lib directory?

Reply:
If we want (2), then we should use filepath.WalkDir() and for each dir that contains a `*.jar` we could and filepath.Join(dir, "*") to the class path. That would include also `lib` (or any other) sub-directory.

Reply:
>Can you give me a reference where there would be a lib directory?

For instance Quarkus https://quarkus.io/ creates `./lib` directory when building in `legacy-jar` mode. For `fast-jar` mode it creates even more sub-dirs.

Reply:
Having those sub-directories is useful for efficient layering. The `./lib` is updated less often than application code so it it's better to have it as separate layers.

Reply:
https://quarkus.io/blog/quarkus-1-12-0-final-released/#fast-jar-as-default

Reply:
Is there a sample project by chance somewhere in GitHub that I can just use?

Reply:
just crate project here https://code.quarkus.io/?e=resteasy